ledger-core/v2/deploy/kube/3-pulsar.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pulsar-config
+  namespace: insolar
+data:
+  pulsard.yaml: |
+    log:
+      level: info
+    pulsar:
+      pulsetime: 10000
+      receivingsigntimeout: 1000
+      receivingnumbertimeout: 1000
+      receivingvectortimeout: 1000
+      receivingsignsforchosentimeout: 0
+      neighbours: []
+      numberdelta: 10
+      distributiontransport:
+        protocol: TCP
+    #    address: 127.0.0.1:58091
+    #    fixedpublicaddress: ""
+      pulsedistributor:
+        bootstraphosts:
+          - virtual-0.platform-network.insolar:13831
+          - virtual-1.platform-network.insolar:13831
+          - virtual-2.platform-network.insolar:13831
+          - virtual-3.platform-network.insolar:13831
+          - virtual-4.platform-network.insolar:13831
+        pingrequesttimeout: 1000
+        randomhostsrequesttimeout: 1000
+        pulserequesttimeout: 1000
+        randomnodescount: 5
+    keyspath: "/etc/insolar/keys/pulsar-0.json" # TODO after https://insolar.atlassian.net/browse/PENV-123
+    tracer:
+      jaeger:
+        collectorendpoint: ""
+        agentendpoint: ""
+        probabilityrate: 1
+    metrics:
+      listenaddress: 0.0.0.0:9090
+      namespace: insolar
+      zpagesenabled: false
+      reportingperiod: 0s
+
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: pulsar
+  namespace: insolar
+spec:
+  selector:
+    matchLabels:
+      app: pulsar
+  replicas: 1
+  serviceName: platform-network
+  template:
+    metadata:
+      labels:
+        app: pulsar
+        role: pulsar
+        scope: platform
+    spec:
+      initContainers:
+        - name: wait
+          image: "bitnami/kubectl:1.18"
```

Comment:
Would love to see #TODO here, since I got zero-trust in bitnami

Reply:
done